angular-portfolio/src/app/bio/bio.component.scss

```diff
@@ -0,0 +1,40 @@
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    height: 75vh;
+}
+
+.picture {
+    flex: 50%;
+    width: 50vw;
+}
+
+.picture img {
+    border-radius: 50%;
+    width: 50%;
+    margin-left: auto;
+    margin-top: 3vh;
+    margin-left: 35%;
+}
+
+.bio {
+    flex: 50%;
+    text-align: center;
+    margin-top: 5%;
+}
+
+p {
+    padding-left: 15%;
```

Comment:
Prefer absolute pixel sizes, particularly for padding and font-sizes. It's difficult to know what this will look like, especially since it could be altered by different screen sizes.